### PR TITLE
[Docs] [#513] Fix code tab selection logic

### PR DESCRIPTION
**Pull request by aarondav**
As reported in #513, clicking the same tab twice closes all tabs. This PR fixes that an a couple other bugs.

### CODE REVIEW

File: docs/source/languagesections/languagesections.js [hunk 1]
```diff
@@ -52,24 +52,27 @@ $(function() {
   $('div.code-section ul.section-selector li,div.plain-section ul.section-selector li').click(function(evt) {
     evt.preventDefault();
 
-    var sel_class = $(this).attr('class').replace(' docutils container', '').replace('notranslate', '');
+    var sel_class = $(this).attr('class')
+      .replace(' docutils container', '')
+      .replace('notranslate', '')
+      .replace(' selected', '');
```
Comment (aarondav) [inline docs/source/languagesections/languagesections.js]: Clicking the same cell currently sees `sel_class = "r selected"`. This gets rid of the selected.

File: docs/source/languagesections/languagesections.js [hunk 2]
```diff
@@ -52,24 +52,27 @@ $(function() {
...
     $('ul.section-selector li').each(function() {
       var parent = $(this).parent().parent();
-      
+      var my_sel_class = sel_class;
       // When the target language is not available, default to bash or python.
       if (!$('div.' + sel_class, parent).length) {
-        if ($('div.' + 'highlight-bash', parent).length)
-          sel_class = 'highlight-bash';
+        if ($('div.bash', parent).length)
+          my_sel_class = 'bash';
```
Comment (aarondav) [inline docs/source/languagesections/languagesections.js]: This was mutating the global-er `sel_class` per iteration loop, so we'd switch permanently to bash/python if available.

File: docs/source/languagesections/languagesections.js [hunk 3]
```diff
@@ -52,24 +52,27 @@ $(function() {
...
         else
-          sel_class = 'highlight-python';
+          my_sel_class = 'python';
```
Comment (aarondav) [inline docs/source/languagesections/languagesections.js]: sel-class should be just `python` or `bash`, not `highlight-*`.

**Review by smurching — APPROVED** [review]
LGTM!

**Pull request merged**